onchain/rollups/test/foundry/inputs/InputBox.t.sol

```diff
@@ -158,13 +158,33 @@ contract InputBoxTest is Test {
         assertEq(inputBox.getNumberOfInputs(_dapp), 0);
     }
 
+    function getMaxInputPayloadLength() internal pure returns (uint256) {
+        // an input blob instance
+        bytes memory blob = abi.encodeWithSignature(
+            "EvmInput(address,uint256,uint256,uint256,bytes)",
+            address(0),
+            0,
+            0,
+            0,
+            new bytes(32)
+        );
+        // number of bytes in input blob excluding input payload
+        uint256 extraBytes = blob.length - 32;
+        // because it's abi encoded, input payloads are stored as multiples of 32 bytes
+        uint256 maxLength = ((CanonicalMachine.INPUT_MAX_SIZE - extraBytes) /
+            32) * 32;
+        return maxLength;
```

Comment:
Why not return this expression right away, instead of storing this in a variable and returning it. Then, maybe it all fits in one line.